app/gui/view/documentation/assets/input.css

```diff
@@ -0,0 +1,3 @@
+@tailwind base;
```

Comment:
> I think the env variable cause should guard it; otherwise, you depend on the node, and again it brings complications If script generates a slightly different output on different machines.

Ok, but then let's put an info here, that the changes in this file, to be effective, require setting proper env. variable.

Reply:
Done